src/app/analysis-types-list/analysis-names-list/analysis-names-list.component.spec.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * analysis-names-list.component.ts
+ * Module for analysis names list.
+ *
+   Copyright 2018 Roman Lisak
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import {MatListModule, MatExpansionModule, MatButtonModule, MatCardModule} from '@angular/material';
+import { AnalysisTypesListComponent } from "../analysis-types-list/analysis-types-list.component";
+import {} from 'jasmine';
```

Comment:
Empty import

Reply:
It was necessary for the test to see _describe_ , _beforeEach_, _it_. Only in this file, don't know why.